herepy/public_transit_api.py

```diff
@@ -32,6 +32,7 @@ def __init__(self,
 
     def __get(self, data, path, json_node):
         url = Utils.build_url(self._base_url + path, extra_params=data)
+        print(url)
```

Comment:
I guess you forgot this line, please remove.

Reply:
Goddamnit. Thank you for catching this !